config/samples/example-test.sh

```diff
@@ -537,29 +537,6 @@ info "5.9.3 Plan a nignx with private chartmuseum(basic auth enabled) "
 kubectl apply -f config/samples/core_v1alpha1_componentplan_mynginx.yaml
 waitComponentPlanDone "kubebb-system" "mynginx" 
 
-info "5.8 Verify that helm add repo with basic auth"
```

Comment:
mabe you can try `git fetch upstream && git checkout upstream/main -- config/sample/example-test.sh` to restore this file.

Reply:
+1

Reply:
@zqq454224016 pls revert changes in this file as I will fix this issue in another PR.

Reply:
These deletions didn't actually skip the #274 ...